### ISSUE TITLE
Typo in recipes/qt6.py

### ISSUE THREAD

**Issue by alfwatt** [ronaldoussoren/py2app#448]
```
$ pip show py2app
Name: py2app
Version: 0.28
...
Location: /opt/homebrew/lib/python3.9/site-packages
Requires: altgraph, macholib, modulegraph
Required-by: 
```

In `recipes/qt6.py`

```
        reslt = {"packages": ["PyQt6"]}
        result.update(extra)
        return result
```

Should read:

```
        result = {"packages": ["PyQt6"]}
        result.update(extra)
        return result
```

**Comment by alfwatt** [comment]
Resolved in commit: 7476aecc555cd9f42701b421e562f992b30f5874

**Comment by alfwatt** [comment]
Which didn't make it into `v0.28` workaround is to install from git directly with pip:

```
$ pip install git+https://github.com/ronaldoussoren/py2app@master
# installation intensifies
$ pip show py2app
Name: py2app
Version: 0.29
Summary: Create standalone macOS applications with Python
Home-page: 
Author: 
Author-email: Ronald Oussoren <ronaldoussoren@mac.com>
License: 
Location: /opt/homebrew/lib/python3.9/site-packages
Requires: altgraph, macholib, modulegraph, packaging, rich, setuptools
Required-by: 
```

**Comment by ronaldoussoren** [comment]
Thanks for the report. I've pushed a fix to a bug fix branch for the 0.28 release, and hope to do a release later today (depends on how much progress I have on #444)
